doc/classes/Object.xml

```diff
@@ -153,18 +181,73 @@
 				A [code]signal[/code] can only be connected once to a [code]method[/code]. It will throw an error if already connected, unless the signal was connected with [constant CONNECT_REFERENCE_COUNTED]. To avoid this, first, use [method is_connected] to check for existing connections.
 				If the [code]target[/code] is destroyed in the game's lifecycle, the connection will be lost.
 				Examples:
-				[codeblock]
-				connect("pressed", self, "_on_Button_pressed") # BaseButton signal
-				connect("text_entered", self, "_on_LineEdit_text_entered") # LineEdit signal
-				connect("hit", self, "_on_Player_hit", [ weapon_type, damage ]) # User-defined signal
-				[/codeblock]
+				[codeblocks]
+				[gdscript]
+				var button = Button.new()
+				button.pressed.connect(_on_Button_pressed) # is the same as:
+				button.connect("pressed", self, "_on_Button_pressed") # BaseButton signal
+
+				var line_edit = LineEdit.new()
+				line_edit.text_entered.connect(_on_LineEdit_text_entered) # is the same as:
+				line_edit.connect("text_entered", self, "_on_LineEdit_text_entered") # LineEdit signal
+
+				var player = Player.new() # assuming you have a Player class that defines the hit signal.
+				var weapon_type = "sword"
+				var damage = 100
```

Comment:
I don't know if I like adding so many extraneous code to make the code "correct" if copy pasted, but when those elements are arbitrary and not necessary to the actual documentation of the current method.

WDYT @godotengine/documentation?

Reply:
I think making the code "correct" is a good idea, but note that there are two ways to do this:

* Declaring `damage` like this.

* Using the literal `100` instead of the variable `damage` in the below lines.

Reply:
Ok. Well I then use the literals directly

Reply:
done